src/providers/AwsProvider.ts

```diff
@@ -1,20 +1,26 @@
-import { App, CfnOutput, Stack } from "@aws-cdk/core";
+import type { CfnOutput } from "@aws-cdk/core";
+import { App, Stack } from "@aws-cdk/core";
 import { get, merge } from "lodash";
-import { AwsCfInstruction, AwsLambdaVpcConfig } from "@serverless/typescript";
+import type { AwsCfInstruction, AwsLambdaVpcConfig } from "@serverless/typescript";
+import type { ProviderInterface } from "@lift/providers";
+import type { ConstructInterface, StaticConstructInterface } from "@lift/constructs";
+import { DatabaseDynamoDBSingleTable, Queue, StaticWebsite, Storage, Vpc, Webhook } from "@lift/constructs/aws";
 import { getStackOutput } from "../CloudFormation";
-import { CloudformationTemplate, Provider as LegacyAwsProvider, Serverless } from "../types/serverless";
-import { awsRequest } from "./aws";
-import { ConstructInterface } from ".";
-import { StaticConstructInterface } from "./Construct";
+import type { CloudformationTemplate, Provider as LegacyAwsProvider, Serverless } from "../types/serverless";
+import { awsRequest } from "../classes/aws";
 import ServerlessError from "../utils/error";
-import { Storage } from "../constructs/Storage";
-import { Queue } from "../constructs/Queue";
-import { Vpc } from "../constructs/Vpc";
-import { Webhook } from "../constructs/Webhook";
-import { StaticWebsite } from "../constructs/StaticWebsite";
-import { DatabaseDynamoDBSingleTable } from "../constructs/DatabaseDynamoDBSingleTable";
-
-export class AwsProvider {
+
+const AWS_DEFINITION = {
+    type: "object",
+    properties: {
+        profile: { type: "string" },
```

Comment:
Isn't the `aws` provider not configurable in the `providers` section?

If yes, that line should probably be removed right?